x-pack/plugins/monitoring/public/services/clusters.js

```diff
@@ -69,9 +69,9 @@ export function monitoringClustersProvider($injector) {
       if (Legacy.shims.isCloud) {
         return Promise.resolve();
       }
-
+      const css = window.location.hash.split('?')[0] === '#/home' ? 'css' : 'single';
```

Comment:
I'm not a big fan of hard-coding a URL when I think we can manage without it.

`monitoringClusters` is only called from a few places, and we can just provide an additional parameter to the function call for `ccs` and we can specify which one from the invokers. 

WDYT?

Reply:
@chrisronline Actually this won't work. I tried passing it here: https://github.com/elastic/kibana/blob/7d8027d0d4697f7c84f202b5661aa135e9899dbb/x-pack/plugins/monitoring/public/views/cluster/listing/index.js#L20 but looks like `monitoringClusters` executes somewhere else before hand (probably in a `routeInit`), which sets the once variable before we can check the param. This means I would have to do some extra logic to determine the "real" origin (not as clean as the one liner above)